.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug Report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+1. Version of F*
+2. Version of KaraMeL
```

Comment:
do we expect bug reports to involve the verification toolchain? or should we ask for which source code version they used (either hacl-packages tagged release, or git revision from this repo)?

Reply:
Could we specify how to get both versions? (fstar.exe --version and krml -version)?